lib/std/os/linux/io_uring.zig

```diff
@@ -404,7 +404,7 @@ pub const IO_Uring = struct {
         sqe.fd = fd;
         sqe.off = offset;
         sqe.addr = @ptrToInt(buffer.ptr);
-        sqe.len = @truncate(u32, buffer.len);
+        sqe.len = @intCast(u32, buffer.len);
```

Comment:
for `read` and `write` these should be `truncate`: they can return partial failures.

Reply:
Done!